zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -2,19 +2,11 @@
   <form class="form-inline" role="form">
     <div class="form-group">
       <select data-placeholder="Service Name" class="form-control input-sm" style="width: 234px;" name="serviceName" id="serviceName">
-        <option value=""></option>
```

Comment:
in our installation, populating service names takes from one to several seconds.  Adrian tried to add caching control some time ago, but it doesn't look like it helped in our case. Does it makes sense to cache the whole HTML of the dropdown in the UI?